client/my-sites/earn/components/commission-fees.tsx

```diff
@@ -0,0 +1,43 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import ExternalLink from 'calypso/components/external-link';
+
+type CommissionFeesProps = {
+	className?: string;
+	commission: number;
+};
+
+const CommissionFees = ( { commission, className }: CommissionFeesProps ) => {
+	const translate = useTranslate();
+
+	return (
+		commission !== null && (
+			<div className={ className }>
+				{ translate(
+					'On your current plan, WordPress.com charges {{em}}%(commission)s{{/em}}.{{br/}} Additionally, Stripe charges are typically %(stripe)s. {{a}}Learn more{{/a}}',
+					{
+						args: {
+							commission: '' + parseFloat( commission ) * 100 + '%',
+							stripe: '2.9%+30c',
+						},
+						components: {
```

Comment:
I think `createInterpolateElement(` can be used. But I am not sure which one is closer to the latest standards....
It's a bit more readable. Cf something like [this](https://github.com/Automattic/wp-calypso/blob/dc483dabe82a9c97bb34f441e2f359a6d68ec32d/client/blocks/import-light/summary/index.tsx#L25)

Reply:
Agreed, gotta switch to `__()`  at least when we redesign the page, but not a blocker now imho.

Reply:
👍